cloud/user-management.md

```diff
@@ -0,0 +1,69 @@
+# User management
+You can manage users for your Timescale Cloud account by navigating to the
+`Members` section in your Cloud console. From here, you can see the current
+members of your Timescale Cloud project, and add and remove members.
+
+## About Timescale Cloud user management
+Timescale Cloud allows you to collaborate with other users on your projects.
+When you have created your project, you can add other users so that they can see
+your project in their Timescale Cloud console. You can add, manage, and delete
+users from the `Members` section of the Cloud console.
+
+If you are the creator of the project, you are assigned an `Administrator` role,
+and have the ability to add and delete other users, as well as change the
+project name. If you have been added to a project, you are assigned a `Member`
+role.
+
+## Add or remove members
+You can add members to your project group from the `Members` section in your
+Cloud console.  The new member does not need to have a Timescale Cloud account
+before you add them. If they do not yet have an account, they are prompted to
+create one.
+
+<procedure>
+
+### Adding members to a project group
+1.  [Log in to your Timescale Cloud account][cloud-login] and navigate to
+    the `Members` section.
+1.  Click `Add New Member`.
+1.  Type the email address of the person that you want to add. A confirmation
+    email is sent to the email address you entered.
+1.  Follow the instructions in the confirmation email to confirm the email
+    address. Alternatively, the new member can create or sign in to a Timescale
+    Cloud account with the email address, and accept the invitation by
+    navigating to the `Invitations` section, and clicking `Accept`.
```

Comment:
Note: If an email that isn't currently a TSCloud member is invited to a project, their way of joining that project would be through receiving the email, clicking "Accept invite" btn, being taken to the sign up page, after signing up, being taken to login page, and when they login for the first time, they're **auto accepted** into said project. First time users don't explicitly accept via Invitations page.